app/js/views/checkrequest.js

```diff
@@ -38,7 +64,7 @@ define(
                 name: "description",
                 label: "Description",
                 control: "textarea",
-                helpMessage: "Please include place of purchase and list of items purchased.  Also submit corresponding recipt."
+                helpMessage: "Please include place of purchase and list of items purchased<br>and submit corresponding recipt."
```

Comment:
If we remove the <br>, does something bad happen?  I can't see wht would go wrong.

Usually we are trying to separate the information (like the sentence) from the display (formatting details like the <br>).  So I kind of cringe when I see HTML stuck in the middle of JavaScript strings.  But this is so minor that I don't really mind.

Reply:
No cringing!! Not acceptable!
I  am not sure what comma you are referring to.
I have the sentence wrong anyhow, it should have parentheses inside the quotes.
I'll take another look at this.

Reply:
Do you want this label separated into two lines?  
Is that why you have it broken up into two strings?  

label=('Description '
             '(Please include place of purchase and list of items purchased '
             'and submit corresponding receipt.)')
  )

I am not sure how to do it *yet* without \<br\>.

This is what I have tried: 

**escaping**
'first     **\n**     string'   =   *string on one line*
'first     **\r**     string'   =   *string on one line*

**concat**
'first '   **+**   'second '  =   *string on one line*
'first '.concat('second')   =   *string on one line*

**comma**
('first '   **,**   'second '  )  =    *error*

Reply:
I will try the comma.  I think it looks better.